Description: fix test for autopkgtest
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-04-28

--- a/test/run.js
+++ b/test/run.js
@@ -26,8 +26,8 @@
   require("./tests-numeric-separators.js");
   require("./tests-class-features-2022.js");
   require("./tests-module-string-names.js");
-  var acorn = require("../acorn")
-  var acorn_loose = require("../acorn-loose")
+  var acorn = require("acorn")
+  var acorn_loose = require("acorn-loose")
 
   var htmlLog = typeof document === "object" && document.getElementById('log');
   var htmlGroup = htmlLog;
--- a/test/tests.js
+++ b/test/tests.js
@@ -4,7 +4,7 @@
 if (typeof exports !== "undefined") {
   var driver = require("./driver.js");
   var test = driver.test, testFail = driver.testFail, testAssert = driver.testAssert;
-  var acorn = require("../acorn");
+  var acorn = require("acorn");
 }
 
 test("import ''", {
